cmd/faucet/README.md

```diff
@@ -50,4 +50,4 @@ Sybil protection via Facebook uses the website to directly download post data th
 
 ## Miscellaneous
 
-Beside the above - mostly essential - CLI flags, there are a number that can be used to fine tune the `faucet`'s operation. Please see `faucet --help` for a full list.
\ No newline at end of file
+Besides the above - mostly essential - CLI flags, there is a number that can be used to fine-tune the `faucet`'s operation. Please see `faucet --help` for a full list.
```

Comment:
"Beside [the flags] above" vs "Besides [the flags] above" -- former seems more correct to me
"there are a number [of flags] that can be used" vs
"there is a number [of flags] that can be used". Again, former seems more correct

Reply:
Ok! Just edited that part